test/verify/test_reverse.cpp

```diff
@@ -0,0 +1,18 @@
+#include "verify_program.hpp"
+#include <migraphx/program.hpp>
+#include <migraphx/generate.hpp>
+#include <migraphx/make_op.hpp>
+
+struct test_reverse : verify_program<test_reverse>
+{
+    migraphx::program create_program() const
+    {
+        migraphx::program p;
+        auto* mm = p.get_main_module();
+        migraphx::shape s{migraphx::shape::float_type, {4, 16}};
+        auto a0  = mm->add_parameter("data", s);
+        int axis = 0;
+        mm->add_instruction(migraphx::make_op("reverse", {{"axis", axis}}), a0);
+        return p;
+    }
+};
```

Comment:
Maybe you can add a test for axis to be a negative number.

Reply:
Added `test/verify/test_reverse_negaxis.cpp`